autopilot/scanner/scanner.go

```diff
@@ -160,10 +185,6 @@ func (s *scanner) scanHosts(ctx context.Context, hs HostScanner, cutoff time.Tim
 	// define worker
 	worker := func(jobs <-chan scanJob) {
 		for h := range jobs {
-			if s.isShutdown() || s.isInterrupted() {
-				return // shutdown
-			}
-
 			scan, err := hs.ScanHost(ctx, h.hostKey, DefaultScanTimeout)
 			if errors.Is(err, context.Canceled) {
```

Comment:
This should now probably also check for the scanner shutting down and the interruption error to avoid logging errors for those.

Reply:
Hm true. I actually thought about that but I figured the `Cause` would wrap `context.Canceled` which is not the case I checked. I'll update it.